src/main/java/seedu/address/model/Model.java

```diff
@@ -123,12 +125,45 @@
     /** Returns an unmodifiable view of the filtered facilitator list */
     ObservableList<Facilitator> getFilteredFacilitatorList();
 
+    /**
+     * Returns true if a module with the same identity as {@code module} exists in Mod Manager.
+     */
+    boolean hasTask(Task module);
+
+    /**
+     * Deletes the given module.
+     * The module must exist in Mod Manager.
+     */
+    void deleteTask(Task target);
+
+    /**
+     * Adds the given module.
+     * {@code module} must not already exist in Mod Manager.
+     */
+    void addTask(Task module);
```

Comment:
The header comments for hasTask, deleteTask, addTask should change from module to task

Reply:
Resolved. Thanks!